fastods/src/main/java/com/github/jferard/fastods/attribute/VerticalAlign.java

```diff
@@ -42,7 +42,12 @@
     /**
      * "top: to the top of the line."
      */
-    TOP("top");
+    TOP("top"),
+    
+    /**
+     * "automatic: automatic. (default)"
+     */
+    AUTOMATIC("automatic");
```

Comment:
The ODF 1.2 and 1.3 specs state (https://docs.oasis-open.org/office/OpenDocument/v1.3/cs01/part3-schema/OpenDocument-v1.3-cs01-part3-schema.html#__RefHeading__1418020_253892949):
> The value of this attribute can be either top, middle, or bottom.

I'm reluctant to this modification. Where did you get this `automatic` value?

Reply:
It was the default by Excel, but it doesnt seem to do much of a difference so top should be fine as default